docs/content/docs/start/tutorial.md

```diff
@@ -7,188 +7,156 @@ aliases:
 
 {{< fluid_bundle_loader idPrefix="dice-roller"
     bundleName="dice-roller.12142020.js" >}}
 
 In the DiceRoller app, users are shown a die with a button to roll it. When the die is rolled, Fluid Framework syncs the data across clients so everyone sees the same result. To do this, complete the following steps:
 
-1. Write the view.
-1. Define the interface the model will expose.
-1. Write the model using the Fluid Framework.
-1. Include the model in a container.
-1. Connect the container to the service for collaboration.
-1. Connect the model instance to the view for rendering.
+1. Application setup
+2. Create a Fluid Container
+3. Write the dice view
+4. Connect view to Fluid data
 
 
-## Write the view
+## Application setup
 
-In this app, you render the view without any UI libraries such as React, Vue, or Angular. Use [TypeScript](https://www.typescriptlang.org/) and HTML/DOM methods. However, Fluid is impartial to how you write your view, so you could use your favorite view framework instead.
+To create a Fluid application that connects to the [Azure Fluid Relay service]({{< relref "azure-frs.md" >}}), start by creating a new instance of the Azure Fluid Relay service client. The client is responsible for creating and loading containers as well as communicating changes between all of the collaborators. Learn more about configuring the client at [Azure Fluid Relay service client]({{< relref "azure-frs.md" >}}).
 
-Since you haven't created your model yet, hardcode a "1" and log to the console when the button is clicked.
+The app creates Fluid containers using a configuration that defines a set of *initial objects* that will be available in the container. Learn more about initial objects in [Data modeling]({{< relref "data-modeling.md" >}}).
 
-```ts
-export function renderDiceRoller(div: HTMLDivElement) {
-    const wrapperDiv = document.createElement("div");
-    wrapperDiv.style.textAlign = "center";
-    div.append(wrapperDiv);
-    const diceCharDiv = document.createElement("div");
-    diceCharDiv.style.fontSize = "200px";
-    const rollButton = document.createElement("button");
-    rollButton.style.fontSize = "50px";
-    rollButton.textContent = "Roll";
+Lastly, `root` defines the HTML element that the Dice will render on.
 
-    rollButton.addEventListener("click", () => { console.log("Roll!"); });
-    wrapperDiv.append(diceCharDiv, rollButton);
-
-    const updateDiceChar = () => {
-        const diceValue = 1;
-        // Unicode 0x2680-0x2685 are the sides of a die (⚀⚁⚂⚃⚄⚅).
-        diceCharDiv.textContent = String.fromCodePoint(0x267F + diceValue);
-        diceCharDiv.style.color = `hsl(${diceValue * 60}, 70%, 50%)`;
-    };
-    updateDiceChar();
-}
-```
+```js
+import { AzureClient } from "@fluidframework/azure-client";
+import { SharedMap } from "fluid-framework";
 
-## Define the model interface
+const client = new AzureClient(localConfig);
```

Comment:
`localConfig` doesn't seem to be defined in this context

Reply:
the previous demo omitted some of the boilerplate code required to set up the actual application. I link to the Azure FRS docs for those needing to set this up on their own.